website/docs/docs/dbt-versions/core-upgrade/05-upgrading-to-v1.10.md

```diff
@@ -51,35 +51,22 @@ You can read more about each of these behavior changes in the following links:
 
 ### Deprecation warnings
 
-New deprecation warnings in v1.10.
-
-#### Duplicate keys in profiles.yml
+Starting in `v1.10`, you will receive deprecation warnings for dbt code that will become invalid in the future, including: 
 
+- Custom inputs (e.g. unrecognized resource properties, configurations, and top-level keys)
+- Duplicate YAML keys in the same file
+- Unexpected jinja blocks (e.g. `{% endmacro %}` tags without a corresponding `{% macro %}` tag)
+- And more
 
-In v1.10, if two identical keys exist in the `profiles.yml`, you will get a warning, and in a future version, dbt will stop supporting duplicate keys with silent overwrite. 
-Previously, if identical keys existed in the [`profiles.yml` file](/docs/core/connect-data-platform/profiles.yml), dbt would use the last configuration listed in the file. 
+We will start raising these warnings in `v1.10`, but making these changes will not be prerequisite to use `v1.10`. We know it will take existing users time to migrate their projects, and it is not our goal to break anyone overnight. The goal is to enable dbt users to work with more safety, feedback, and confidence going forward.
 
-```yml
+What does this mean for you?
 
-my_profile:
-  target: 
-  outputs:
-...
-
-my_profile: # dbt would use this profile key
-  target: 
-  outputs:
-...
-
-```
+1. If your project (or dbt package) hits a new deprecation warning in `v1.10`, you should plan to update your invalid code soon! Even though it’s just a warning for now, in a future version of dbt, we will be enforcing stricter validation to the inputs of your dbt project.
```

Comment:
I think we should add a callout as well to the tool @b-per is working on (https://github.com/dbt-labs/dbt-cleanup) that you can use to autofix a lot of these.

Open to ideas on wording!

Reply:
I'll check with Benoit about the tools status and circle back to this